### ISSUE TITLE
Game Launcher Minor Issue

### ISSUE THREAD

**Issue by RaidenRafael** [an-anime-team/the-honkers-railway-launcher#102]
Hi, So I Am Only Here To Create A Small Issue Ticket About The Launcher, As Listed Below

1: The Launcher Thinks The Game Fails To Launch, Even Tho The Game Launched
2: Since The Launcher Is Thinking The Game Failed To Launch, it Doesn't Save Session (In Game Graphics Settings, Account Login), Kinda Annoying for The long run

That's It Really, Didn't Bother Creating A Support Ticket On The Discord Server Since The Issue Is Small And Can Be Ignored, Installed The Package Via (YAY - AUR RESPOSITORY), No Other Issues With In Game Graphics.

Thank You

**Comment by RaidenRafael** [comment]
![image](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/67449286/aa71eb96-dc1d-4dab-a479-ea4137115e26)
I Managed To Catch The Error Of It While The Game Is Still Running

**Comment by HxxThSwggr** [comment]
Same issue, but the game does not work. I can hear menu music, but the screen is black.

**Comment by EnergoStalin** [comment]
Perhaps related https://discord.com/channels/910869215857217596/1214423720886669322 not sure it related but i guess it uses same core with general purpose launcher for checking.

TLDR: Try adding os.execute('sleep 10') at the start of v1_game_is_running in integration.lua and check if it's fixed.
